server.py

```diff
@@ -0,0 +1,82 @@
+#! /usr/bin/env python3
+
+# Author: Thomas Gruber
+# Version: 0.8, 11/06/18
+
+from http.server import HTTPServer, BaseHTTPRequestHandler
+
+import io
+import json
+import time
+
+
+class ControlHandler (BaseHTTPRequestHandler):
+    SERVER_PATH = '/terradrone'
+    CHAR_ENCODING = 'utf-8'
+
+    # commands should correspond to those written in Android application
+    CMD_ERROR                                                   = 0
+    CMD_TEST                                                    = 1
+    CMD_FORWARD_START                                           = 100
+    CMD_FORWARD_STOP                                            = 101
+    CMD_BACKWARD_START                                          = 102
+    CMD_BACKWARD_STOP                                           = 103
+    CMD_ROTATE_RIGHT_START                                      = 104
+    CMD_ROTATE_RIGHT_STOP                                       = 105
+    CMD_ROTATE_LEFT_START                                       = 106
+    CMD_ROTATE_LEFT_STOP                                        = 107
+    CMD_COLLECT                                                 = 200
```

Comment:
I know this is just coding style but do those need to be tabbed so far over? Tbh I did not know this would run without errors in python because of how important spacing is in python syntax.

Reply:
I tabbed them over so far to leave room for other (longer worded) commands. Since we only plan on these so far, it's not necessary to have them over. I do prefer them to be lined up though.

Reply:
Commit  3122d08b84f914cacb9d71c4dc32b8d1b5c60c8d decreased tab spaces between variables and their initialization. Still kept numbers lined up correctly.